README.md

```diff
@@ -1,41 +1,73 @@
-# A starter template for Run On Slack apps
+# Deno Starter Template for Run On Slack apps
 
-This repo is a starter template for new Run On Slack apps. 
+This repo is a starter template for new Run On Slack apps. This template uses Deno and the Slack CLI.
 
-## Prerequisites
+Before getting started, make sure you have a development workspace where you have permissions to install apps. If you don’t have one set up, go ahead and [create one](https://slack.com/create).
+
+## Template Versions
+
+#### Accessing Different CLI Versions of Templates
+This template is compatible with version(s) of the Slack CLI.
+
+To view a template that is compatible with a specific version of the CLI, view the list below:
+
+- **(current)**[deno-starter-template for CLI v1.0.0](https://github.com/slack-samples/deno-starter-template/tree/main)
+
+## Installation
+
+#### Prerequisites
 
 To use this template, you will need to have installed and configured the Slack CLI. 
 Do this by following our [Quickstart Guide](https://api.slack.com/future/quickstart).
 
-# Making an app
+You will also need a `slack.json` file - you can learn more about this file in the `Project Structure` section.
```

Comment:
I don't think this line is necessary since this file is already part of the code they're pulling down